packages/shared/styles/components/atoms/SfBreadcrumbs.scss

```diff
@@ -1,37 +1,42 @@
 @import "../../helpers";
 
 .sf-breadcrumbs {
-  font: var(
+  @include font(
     --breadcrumbs-font,
-    font(
-      --breadcrumbs,
-      400,
-      var(--font-size-small),
-      1.6,
-      var(--body-font-family-secondary)
-    )
+    var(--font-weight-regular),
+    var(--font-size-small),
+    1.6,
+    var(--body-font-family-secondary)
   );
   &__list {
+    padding: var(--breadcrumbs-list, var(--spacer-small));
     display: flex;
-    padding: 0;
-    margin: 0;
+    flex-wrap: wrap;
     list-style: none;
   }
   &__item {
+    &:first-child {
+      padding: var(
```

Comment:
I think first-child shouldn't have padding on the left side :)

Reply:
deleted :)